pkg/controller/template/test_data/templates/master/00-master/aws/files/-etc-kubernetes-ca.crt

```diff
@@ -1,6 +1,8 @@
 contents:
-  source: data:,dummy%20root-ca%0A
+  source: data:text/plain,dummy%20root-ca%0A
   verification: {}
-filesystem: root
+group: {}
```

Comment:
this doesn't hurt but isn't required

Reply:
we discovered this is a golang encoding/json quirk.